### PR TITLE
Release 1.3.0

### PR DESCRIPTION
**Pull request by rinjac**

### CODE REVIEW

**Review by andrey999333 — APPROVED** [review]

File: examples/file_transcription_example.py [hunk 1]
```diff
@@ -3,36 +3,23 @@
...
-    channel = grpc.insecure_channel(CHANNEL, options=options)
-    stub = speech_to_text_pb2_grpc.Speech2TextStub(channel=channel)
-
-    return stub
-
-
 def main():
-    stub = create_stub()
+    parser = argparse.ArgumentParser(description='File transcription example.')
+    parser.add_argument("--config", type=str)
```
Comment (AzzerAce) [inline examples/file_transcription_example.py]: can we add descriptions to these arguments?

**Review by AzzerAce — CHANGES_REQUESTED** [review]
can we also add the secure_grpc.json example with placeholder values and the comment i added?
otherwise looks ok

File: examples/file_transcription_example.py [hunk 2]
```diff
@@ -3,36 +3,25 @@
...
-    channel = grpc.insecure_channel(CHANNEL, options=options)
-    stub = speech_to_text_pb2_grpc.Speech2TextStub(channel=channel)
-
-    return stub
-
-
 def main():
-    stub = create_stub()
+    parser = argparse.ArgumentParser(description='File transcription example.')
+    parser.add_argument("--config", type=str, help="The GRPC configuration file. "
```
Comment (AzzerAce) [inline examples/file_transcription_example.py]:
```suggestion
    parser.add_argument("--config", type=str, help="The GRPC configuration file path. "
```

**Review by AzzerAce — CHANGES_REQUESTED** [review]
minor change

**Review by AzzerAce — APPROVED** [review]

**Pull request merged**